### ISSUE TITLE
Getting unknown property exception thrown.

### ISSUE THREAD

**Issue by soap** [mdmsoft/yii2-ar-behaviors#4]
When I use QueryScopeBahavior in my module's model class. I got the following error.

Getting unknown property: frontend\modules\product\models\ProductSearch::modelClass

In D:\yii2\soap\xorders\vendor\yiisoft\yii2\base\Component.php.

My environment is PHP5.5 and Yii2.0.9 Windows 10.

Any recommendation? Thanks in advance.

**Comment by mdmunir** [comment]
when you not using `QueryScopeBahavior`, is it going ok?

**Comment by soap** [comment]
Yes, it happen in the method hasMethod() of QueryScopeBehavior

**Comment by mdmunir** [comment]
`QueryScopeBehavior` only work for ActiveRecord.

**Comment by soap** [comment]
Yes, I know that. But ProductSearch is extended from Product which is inherited from ActiveRecord.

**Comment by mdmunir** [comment]
I mean ActiveQuery.
